tests/pass/iter.rs

```diff
@@ -1,5 +1,6 @@
 fn iter_empty_and_zst() {
-    for _ in Vec::<u32>::new().iter() { // this iterates over a Unique::empty()
+    // Iterate over a Unique::empty()
+    for _ in Vec::<u32>::new().iter() {
```

Comment:
Reworded for consistency with the comment on the other `for` loop after this one.